Emby.Drawing.Skia/SkiaEncoder.cs

```diff
@@ -548,17 +548,17 @@ public string EncodeImage(string inputPath, DateTime dateModified, string output
                 using (var resizedBitmap = new SKBitmap(width, height))//, bitmap.ColorType, bitmap.AlphaType))
                 {
                     // scale image
-                    var resizeMethod = SKBitmapResizeMethod.Lanczos3;
-
-                    bitmap.Resize(resizedBitmap, resizeMethod);
+                    bitmap.ScalePixels(resizedBitmap, SKFilterQuality.High);
 
                     // If all we're doing is resizing then we can stop now
                     if (!hasBackgroundColor && !hasForegroundColor && blur == 0 && !hasIndicator)
                     {
                         _fileSystem.CreateDirectory(_fileSystem.GetDirectoryName(outputPath));
                         using (var outputStream = new SKFileWStream(outputPath))
                         {
-                            resizedBitmap.Encode(outputStream, skiaOutputFormat, quality);
+                            SKImageInfo imageInfo = new SKImageInfo(width,height);
+                            var pixmap = new SKPixmap(new SKImageInfo(width, height), resizedBitmap.GetPixels());
```

Comment:
`SKPixmap` implements `IDisposable`, maybe wrap it in a `using` statement?

Reply:
It should dispose immediately after the return anyway, but it's probably nicer to do it explicitly.